manifests/config.pp

```diff
@@ -0,0 +1,73 @@
+# @summary Manages the configuration of Gitlab runner
+#
+# @api private
+#
+class gitlab_ci_runner::config (
+  $config_path,
```

Comment:
the default values for those parameters need to come from the init.pp, like this:
```suggestion
  $config_path = gitlab_ci_runner::config_path,
```

Reply:
A default value isn't required because the class is private. The value for those parameters will be configured by the init.pp.